src/types/UiaTextRangeBase.cpp

```diff
@@ -214,8 +140,8 @@ HRESULT UiaTextRangeBase::RuntimeClassInitialize(const UiaTextRangeBase& a) noex
     _pProvider = a._pProvider;
     _start = a._start;
     _end = a._end;
-    _degenerate = a._degenerate;
     _pData = a._pData;
+    _wordDelimiters = a._wordDelimiters.c_str();

```

Comment:
This makes me uneasy about capturing someone else's pointer. What if `a` is destroyed before `this` is destroyed? I think the only way this is actually safe is if `_wordDelimiters` is of a type that will take the captured pointer and copy the value out of it before returning. But I can't see what that is right now because this file is a 2,145 line diff that is making Edge cry when I type/scroll.

Reply:
https://github.com/microsoft/terminal/pull/4018/files#r372690478
Removing `.c_str()`

Reply:
sorry, you went _back_ to c_str? what is the exact error in C26xx